<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="org.springframework.transaction.interceptor.RollbackRuleTests" tests="9" skipped="0" failures="0" errors="0" timestamp="2020-12-11T19:53:35" hostname="DESKTOP-TMNPDT1" time="0.043">
  <properties/>
  <testcase name="ctorArgMustBeAThrowableClassWithNonThrowableType()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.001"/>
  <testcase name="ancestry()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.0"/>
  <testcase name="alwaysTrueForThrowable()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.001"/>
  <testcase name="foundImmediatelyWithString()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.001"/>
  <testcase name="ctorArgExceptionStringNameVersionWithNull()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.001"/>
  <testcase name="foundEnclosedExceptionWithEnclosingException()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.003"/>
  <testcase name="ctorArgMustBeAThrowableClassWithNullThrowableType()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.026"/>
  <testcase name="foundImmediatelyWithClass()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.0"/>
  <testcase name="notFound()" classname="org.springframework.transaction.interceptor.RollbackRuleTests" time="0.0"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
